docs/guides/contracts.md

```diff
@@ -52,19 +52,20 @@ Note:
 
 ## 3. Initialize the contract instance
 
-### By sourcecode
+### By source code
 
 ```js
-const CONTRACT_SOURCE = ... // source code of the contract
-const contractInstance = await aeSdk.getContractInstance({ source: CONTRACT_SOURCE })
+const CONTRACT_SOURCE_CODE = ... // source code of the contract
+const contractInstance = await aeSdk.getContractInstance({ source: CONTRACT_SOURCE_CODE.trim() })
+// trim function here is a helper to avoid indentation problems when you directly use source code
```

Comment:
When I was redoing tutorials, I was struggling with formatting.
Then I found this hint in codebase which ultimately solved my problem.
https://github.com/aeternity/aepp-sdk-js/blob/40cc9337888454ca1d8c0e8fd25c013ae3497834/examples/browser/aepp/src/Contracts.vue#L88
So I thought we can pass this hint to other developers .]

Reply:
I see that using `.trim()` is not necessary.  But I got stuck for a while on this when I was redoing the tutorial.
In some tutorials was not needed to trim.


 My motivation is to make redoing examples a smooth experience.
 So adding `trim` is more about being on the safe side.
Somehow we should pass this knowledge to user.

What do you suggest then?

Reply:
I understand better now.
You are talking about loading from file / directly inserted contract source code to js.

But the question is still the same? Any idea how would we elegantly advise users in order to prevent them from being stuck?

Reply:
In which case user would stuck without `trim`?
As I see the contract defined as:
```
const contractSource = `

     contract Test =
         entrypoint getArg(x : map(string, int)) = x
         entrypoint getArg2(x : map(string, int)) = x

`;
```
works fine without `trim`

Reply:
I think it was the case in directly entered contract code in `4. Deploy the contract`. You have to manually adjust white spacing it indenting.
There is the usage of trim function in the codebase in more places for some reason, so I thought we could pass this knowledge to developers.
I have removed the trim function then